typescript/src/shared/configuration.ts

```diff
@@ -11,7 +11,8 @@ export type Object =
   | 'products'
   | 'prices'
   | 'balance'
-  | 'refunds';
+  | 'refunds'
+  | 'search';
```

Comment:
I think this should be `search_documentation` , to avoid confusion with the Search API https://docs.stripe.com/search